js/private/js_run_devserver.mjs

```diff
@@ -6,11 +6,18 @@ import * as child_process from 'node:child_process'
 import * as crypto from 'node:crypto'
 import * as readline from 'node:readline'
 
+// Environment constants
+const {
+    JS_BINARY__EXECROOT,
```

Comment:
Some of the `process.env.JS_BINARY__LOG_DEBUG` lines were actually showing up in the profiling, so may as well make them `const` 🤷